src/ui/components/Account/AccountsList.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import {FlatList, View} from 'react-native';
+import {View} from 'react-native';
 import {Divider} from '@ui/library';
 import type {AccountInfo} from 'src/api/hooks/useAccount';
 import {NavigationProp, useNavigation} from '@react-navigation/native';
 import {AppStackParamList} from '@ui/navigation/navigation';
 import {accountScreen} from '@ui/navigation/routeKeys';
 import {AccountTeaser} from './AccountTeaser';
 import globalStyles from '@ui/styles';
+import {FlashList} from '@shopify/flash-list';
```

Comment:
let's export this from the @library, it will make it easier later if we create an abstraction for lists